src/sdk/namespace_cache.js

```diff
@@ -157,18 +168,32 @@ class NamespaceCache {
     /////////////////
 
     async list_objects(params, object_sdk) {
-        // TODO listing from cache only for deevelopment
-        return this.namespace_nb.list_objects(params, object_sdk);
+        const get_from_cache = params.get_from_cache;
+        params = _.omit(params, 'get_from_cache');
+        if (get_from_cache) {
+            return this.namespace_nb.list_objects(params, object_sdk).then(res => {
+                _.forEach(res.objects, obj => {
+                    obj.create_time = obj.last_modified_time ? obj.last_modified_time : obj.create_time;
```

Comment:
Why is this needed? Shouldn't that be in our s3 level like we do in `s3_utils.set_response_object_md()`?

Reply:
The object listing does not go through `s3_utils.set_response_object_md()`

Reply:
ok, but it seems that this handling is needed in s3_get_bucket instead of just here in namespace_cache.